<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}

QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 5px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}

QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 5px;
}

QLineEdit:focus {
	border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}

QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #24A1DE;
  padding: 5px 15px;
  outline: 0px;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 1px solid #9ac3fe;
}

QProgressBar {
    border: 1px solid grey;
    border-radius: 8px;
	text-align: center;
}

QProgressBar::chunk {
	background-color: #0d6efd;
	border-radius: 8px;
}

QScrollBar:horizontal {
	border: none;
	background-color: #fff;
	width: 2px;
}

QScrollBar::handle:horizontal {
	background-color: #24A1DE;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
	width: 0px;
    subcontrol-origin: margin;
}

QScrollArea {
	border-color: grey;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>671</width>
      <height>287</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>21</y>
        <width>5</width>
        <height>255</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>5</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>21</y>
        <width>459</width>
        <height>90</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Ссылка на пост</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="post_link_edit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>451</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>430</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>52</y>
         <width>451</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="text">
           <string>Начать</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>656</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>21</y>
        <width>180</width>
        <height>218</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>SESSION NAME</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="addAccount_btn">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Добавить аккаунт</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="LOGIN_edit">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>cooluser</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="PHONE_edit">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>+123456789</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>87</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>PHONE</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scroll">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>120</y>
        <width>451</width>
        <height>121</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="container">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>428</width>
         <height>119</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>19</y>
          <width>431</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="check_layout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>100</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Созданные сессии:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>246</y>
        <width>451</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="selectAll_btn">
         <property name="text">
          <string>Выбрать всё</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearAll_btn">
         <property name="text">
          <string>Очистить всё</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reload_btn">
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="accountList_page">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>52</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Список</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="help_page">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>399</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Как добавить аккаунт?</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>0</y>
      <width>651</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Telefarm - ферма телеграмм акаунтов</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LOGIN_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>PHONE_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PHONE_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>addAccount_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>post_link_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>start_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
